populus/api/project.py

```diff
@@ -0,0 +1,77 @@
+import logging
+import os
+import shutil
+
+from .config import (
+    load_user_config,
+)
+
+from populus.config.defaults import (
+    load_default_project_config,
+)
+
+from populus.config.helpers import (
+    check_if_project_json_file_exists,
+    write_project_config,
+    get_project_json_config_file_path,
+)
+
+from populus.utils.filesystem import (
+    ensure_path_exists,
+)
+
+from populus import ASSETS_DIR, Project
```

Comment:
nitpick:

To be consistent with import style from the rest of the project, split these across multiple lines.